__test__/index.test.js

```diff
@@ -0,0 +1,31 @@
+
+import request from 'supertest';
+import app from '../server';
+
+
+let server;
+let agent;
```

Comment:
what if we rename this to `http`, so it reads nicely with the test `await http.get('/')`

Reply:
okay, will do that